lib/sablon/content.rb

```diff
@@ -51,7 +52,7 @@ def initialize(value)
         super value.to_s
       end
 
-      def append_to(paragraph, display_node)
+      def append_to(paragraph, display_node, _env)
```

Comment:
I don't think this codebase uses the `_` prefix to mark unused variables. I'm personally not too big of a fan of it. Can we remove it?